sdlib/d/gc/tcache.d

```diff
@@ -374,9 +374,17 @@ unittest makeRange {
 }
 
 unittest getCapacity {
+	void setUsedCapLarge(void* ptr, size_t usedCapacity) {
+		assert(ptr !is null);
+		auto pd = threadCache.getPageDescriptor(ptr);
+		assert(pd.extent !is null);
+		assert(pd.extent.isLarge());
+		pd.extent.setUsedCapacity(usedCapacity);
+	}
+
 	// Test capacity for non appendable allocs.
-	auto nonAppendable = threadCache.alloc(100, false);
-	assert(threadCache.getCapacity(nonAppendable[0 .. 100]) == 0);
+	auto nonAppendable = threadCache.alloc(8, false);
+	assert(threadCache.getCapacity(nonAppendable[0 .. 8]) == 0);
```

Comment:
Why not 100 anymore?

Reply:
8 is a size class, so even if this was appendable, you should expect 0 capacity. This is not testing anything anymore.